overlord/snapstate/autorefresh.go

```diff
@@ -157,8 +157,14 @@ func canRefreshOnMeteredConnection(st *state.State) (bool, error) {
 	if err != nil && err != state.ErrNoState {
 		return false, err
 	}
-
-	return onMetered != "hold", nil
+	switch onMetered {
+	case "hold":
+		return false, nil
+	case "force":
```

Comment:
`"always"`?

Reply:
I like this, happy to change it. The original name was suggested by @niemeyer so I would like to get his input on this before changing anything.